app/build.gradle.kts

```diff
@@ -31,16 +32,36 @@ android {
         }
 
         release {
+            if (System.getenv("signingKey") != null) {
+                signingConfigs {
+                    create("release") {
+                        storeFile = file(System.getenv("signingKey"))
+                        storePassword = System.getenv("keyStorePassword")
+                        keyAlias = System.getenv("keyAlias")
+                        keyPassword = System.getenv("keyPassword")
+                    }
+                }
```

Comment:
Environment variables generally use screaming snake case. Is something preventing that?

Reply:
Our CI environment has variables named that way. We can't switch unless it's renamed.